bin/docker/Dockerfile

```diff
@@ -1,9 +1,12 @@
-FROM java:openjdk-7-jre
+FROM java:openjdk-7-jre-alpine
 
-ENV LC_ALL C
-ENV LANG C.UTF-8
-ENV DEBIAN_FRONTEND noninteractive
-ENV DEBCONF_NONINTERACTIVE_SEEN true
+# need bash
+RUN apk add --update bash
+
+# fix broken cacerts
+RUN apk add --update java-cacerts && \
+    rm -f /usr/lib/jvm/default-jvm/jre/lib/security/cacerts && \
+    ln -s /etc/ssl/certs/java/cacerts /usr/lib/jvm/default-jvm/jre/lib/security/cacerts
```

Comment:
This is still necessary as the `java:openjdk-7-jre-alpine` still suffers from alpine's busted `cacerts`.